packages/mobile/src/send/QRCodeIcon.tsx

```diff
@@ -7,7 +7,7 @@ import { Screens } from 'src/navigator/Screens'
 
 export function QRCodeIcon() {
   const onQrCodePress = React.useCallback(() => {
-    navigate(Screens.QRScanner)
+    navigate(Screens.QRNavigator, { screen: Screens.QRScanner })
```

Comment:
Is `screen` the default screen here? If so, maybe rename it?

Reply:
No, default screen is `QRCode`